SU-SWS/stanford_basic
# D8CORE-856 Display paragraph contextual links

# READY FOR REVIEW

# Summary
- Display contextual links on paragraphs if they are present (paragraphs_edit module)

# Need Review By (Date)
- 12/11

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_basic:dev-D8CORE-856-paragraph-contextual-links su-sws/stanford_profile:dev-D8CORE-856-paragraph-contextual-links --prefer-source`
1. do an install or `drush cim`
1. on any page with paragraphs, verify you see multiple contextual links
1. click on the far right contextual link (if there are multiple) and click "Edit"
1. make change, save and verify the changes are displayed.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
